beacon-chain/rpc/eth/node/node.go

```diff
@@ -266,11 +267,23 @@ func (ns *Server) GetSyncStatus(ctx context.Context, _ *emptypb.Empty) (*ethpb.S
 	defer span.End()
 
 	headSlot := ns.HeadFetcher.HeadSlot()
+
+	headState, err := ns.HeadFetcher.HeadState(ctx)
+	if err != nil {
+		err = status.Errorf(codes.Internal, "Could not get head state: %v", err)
+	}
+
+	isOptimistic, err := rpchelpers.IsOptimistic(ctx, headState, ns.OptimisticModeFetcher)
+	if err != nil {
+		err = status.Errorf(codes.Internal, "Could not check optimistic status: %v", err)
+	}
+
 	return &ethpb.SyncingResponse{
 		Data: &ethpb.SyncInfo{
 			HeadSlot:     headSlot,
 			SyncDistance: ns.GenesisTimeFetcher.CurrentSlot() - headSlot,
 			IsSyncing:    ns.SyncChecker.Syncing(),
+			IsOptimistic: isOptimistic,
```

Comment:
There is no `IsOptimistic` field in the protocol buffer type definition. Reading https://github.com/prysmaticlabs/documentation/pull/410/files might help you figure out what to do.